DependencyInjection/SensioFrameworkExtraExtension.php

```diff
@@ -46,24 +46,8 @@ public function load(array $configs, ContainerBuilder $container)
             ));
         }
 
-        if ($config['request']['converters']) {
-            $annotationsToLoad[] = 'converters.xml';
-
-            if (class_exists('Symfony\Component\ExpressionLanguage\ExpressionLanguage')) {
-                $container->setAlias('sensio_framework_extra.converter.doctrine.orm.expression_language', new Alias('sensio_framework_extra.converter.doctrine.orm.expression_language.default', false));
-            } else {
-                $container->removeDefinition('sensio_framework_extra.converter.doctrine.orm.expression_language.default');
-            }
-
-            $this->addClassesToCompile(array(
-                // cannot be added because it has some annotations
-                //'Sensio\\Bundle\\FrameworkExtraBundle\\Configuration\\ParamConverter',
-                'Sensio\\Bundle\\FrameworkExtraBundle\\EventListener\\ParamConverterListener',
-                'Sensio\\Bundle\\FrameworkExtraBundle\\Request\\ParamConverter\\DateTimeParamConverter',
-                'Sensio\\Bundle\\FrameworkExtraBundle\\Request\\ParamConverter\\DoctrineParamConverter',
-                'Sensio\\Bundle\\FrameworkExtraBundle\\Request\\ParamConverter\\ParamConverterInterface',
-                'Sensio\\Bundle\\FrameworkExtraBundle\\Request\\ParamConverter\\ParamConverterManager',
-            ));
+        if ($config['request']['resolvers']) {
+            $annotationsToLoad[] = 'resolvers.xml';
```

Comment:
What if you want to use the `@Arg` feature without using the default set of resolvers? If I'm not using doctrine nor the datetime resolver in a project, I don't really need them but would like to expand on the feature.